targets/python/source/source/PlayFabSettings.py

```diff
@@ -0,0 +1,26 @@
+ProductionEnvironmentURL = "https://{titleId}.playfabapi.com{methodUrl}"
+TitleId = "" # You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website)
+GlobalErrorHandler = None
+EntityToken = None # Internal variable used for Entity API Access (basically Entity Login)
+DeveloperSecretKey = None # You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website)
+ClientSessionTicket = None # This is set
+SdkVersionString = "PythonSdk-<%- sdkVersion %>";
+SdkVersionString = ""
```

Comment:
You just clobbered the good SdkVersionString on the line above.
Delete second assignment of SdkVersionString